overlays/holo-nixpkgs/hpos-led-manager/src/main.rs

```diff
@@ -29,18 +32,48 @@ struct Args {
     flag_state: PathBuf,
 }
 
-fn main() -> Fallible<()> {
+#[tokio::main]
+async fn get_hydra_revision() -> anyhow::Result<serde_json::Value> {
```

Comment:
I think it's more logical to annotate `fn main` with `#[tokio::main]`.
```suggestion
async fn get_hydra_revision() -> anyhow::Result<serde_json::Value> {
```